src/sentry/services/hybrid_cloud/user.py

```diff
@@ -12,9 +14,33 @@
 from sentry.silo import SiloMode
 
 
+@dataclass(frozen=True)
+class APIUser:
+    id: int = -1
+    name: str = ""
+    email: str = ""
+    username: str = ""
+    actor_id: int = -1
+
+    # TODO: Extract to base dataclass?
```

Comment:
Interestingly, I added this into the base service.  Checkout `_serialize_organization`.  But that was because I needed some additional custom logic.  I have no opinion beyond that.